examples/fit_models.py

```diff
@@ -120,21 +127,39 @@ def train_model(hparams):
             save_file=os.path.join(hparams['tt_version_dir'], 'train_curves'),
             format='png',
         )
-        plot_training_curves(
-            metrics_file=os.path.join(hparams['tt_version_dir'], 'metrics.csv'),
-            dtype='val',
-            expt_ids=hparams['expt_ids'],
-            save_file=os.path.join(hparams['tt_version_dir'], 'val_curves'),
-            format='png',
-        )
+        
+        if data_gen.n_tot_batches['val'] > 0:
+            plot_training_curves(
+                metrics_file=os.path.join(hparams['tt_version_dir'], 'metrics.csv'),
+                dtype='val',
+                expt_ids=hparams['expt_ids'],
+                save_file=os.path.join(hparams['tt_version_dir'], 'val_curves'),
+                format='png',
+            )
+            
+           
+            plot_train_val_curves(
+                metrics_file=os.path.join(hparams['tt_version_dir'], 'metrics.csv'),
+                expt_ids=hparams['expt_ids'],
+                save_file=os.path.join(hparams['tt_version_dir'], 'train_val_curves'),
+                format='png',
+            )
```

Comment:
what does this function do? it's not imported anywhere. seems like it can be deleted